Samples/CheckMalformedEventsEH/GetMalformedEvents.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+
+namespace CheckMalformedEvents
+{
+    class GetMalformedEvents
+    {
+        //static string connectionString = "HostName=xxxxx.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=xxxxxx";
+        static string connectionString;
```

Comment:
As a sample, this is ok. But there are few issues with this code, variables can be at a lower scope. Error handling can be better.

Reply:
@vigneshcmsft I will merge this PR as the CSS team is interested to get the public doc link soon, will work on your comments shortly.